cmake/detect-intrinsics.cmake

```diff
@@ -536,3 +536,21 @@ macro(check_vgfma_intrinsics)
         HAVE_VGFMA_INTRIN FAIL_REGEX "not supported")
     set(CMAKE_REQUIRED_FLAGS)
 endmacro()
+
+macro(check_xsave_intrinsics)
+    if(NOT NATIVE_FLAG)
```

Comment:
Should be `if (NOT NATIVEFLAG AND NOT MSVC)`?

Reply:
Possibly... On MSVC it would just fail and not set the flag...

Reply:
We can just return() on MSVC then.

Reply:
It doesn't hurt to let MSVC test.... Would not be the first time Microsoft messes up with the intrinsics headers and breaks things... One improvement would be to add preprocessor flag so that on non-Windows platforms, it would also use the intrinsic function and not inline assembly... But some gcc versions are known to have buggy intrinsic function...